workspace/global_init_scripts.go

```diff
@@ -46,10 +46,15 @@ type globalInitScriptCreateResponse struct {
 	ScriptID string `json:"script_id"`
 }
 
+type globalInitScriptListResponse struct {
+	Scripts []GlobalInitScriptInfo `json:"scripts"`
+}
+
 // List returns a list of registered global init scripts
-func (a GlobalInitScriptsAPI) List() (initScripts []GlobalInitScriptInfo, err error) {
-	err = a.client.Get(a.context, "/global-init-scripts", nil, &initScripts)
-	return
+func (a GlobalInitScriptsAPI) List() ([]GlobalInitScriptInfo, error) {
```

Comment:
why don't you like named go arguments? :) https://tour.golang.org/basics/7

this will save one line not to be covered by tests